tox.ini

```diff
@@ -42,16 +42,10 @@ commands =
 [docs]
 docs_dir = {tox_root}{/}docs
 build_dir = {[docs]docs_dir}{/}_build
-# replace when Sphinx>=7.3 and Python 3.8 is dropped:
-#  -T => --show-traceback
-#  -W => --fail-on-warning
-#  -b => --builder
-#  -v => --verbose
-#  -a => --write-all
-#  -E => --fresh-env
-sphinx_args = -T -W --keep-going --jobs auto
+sphinx_args = --show-traceback --fail-on-warning --keep-going --jobs auto
 
 [testenv:docs{,-lint,-all,-live,-live-src}]
```

Comment:
And as we [did with Toga](https://github.com/beeware/toga/commit/4ea3c24c4b5241abb056c760db1e218161c49e0f), we should add `suicide_timeout` for running lived docs:
```
# give sphinx-autobuild time to shutdown http server
suicide_timeout = 1
```
Any other repo supporting a live view would benefit as well.